.buildkite/pipeline.deploy.yml

```diff
@@ -8,8 +8,13 @@ steps:
 
   - label: ":docker: :rocket: Latest"
     plugins:
-      - aws-assume-role-with-web-identity:
+      - aws-assume-role-with-web-identity#v1.4.0:
           role-arn: arn:aws:iam::445615400570:role/pipeline-buildkite-plugin-tester-deploy
+          session-tags:
+            - organization_slug
+            - organization_id
+            - pipeline_slug
+            - build_branch
```

Comment:
From reading the code, it doesn't look like the `build_branch` tag is required by the role's trust policy. But I don't think it breaks anything if it's here.

Reply:
That's right, it shouldn't break. And these steps are limited to if build.branch == main (L27) so I'm keen to pass it in here, and then restrict the trust policy as a secondary step.